apps/condo/lang/en/en.json

```diff
@@ -1,1473 +1,1474 @@
-  "excelExport.headers.ticket_report_status_categoryClassifier.closed": "Closed",
-  "excelExport.headers.ticket_report_status_categoryClassifier.new_or_reopened": "Opened",
-  "excelExport.headers.ticket_report_status_dayGroup.address": "Address",
-  "excelExport.headers.ticket_report_status_dayGroup.date": "Date",
-  "excelExport.headers.ticket_report_status_dayGroup.processing": "Processing",
-  "excelExport.headers.ticket_report_status_dayGroup.completed": "Completed",
-  "excelExport.headers.ticket_report_status_dayGroup.canceled": "Canceled",
-  "excelExport.headers.ticket_report_status_dayGroup.deferred": "Deferred",
-  "excelExport.headers.ticket_report_status_dayGroup.closed": "Closed",
-  "excelExport.headers.ticket_report_status_dayGroup.new_or_reopened": "New or reopened",
-  "excelExport.headers.ticket_report_status_executor.executor": "Executor",
-  "excelExport.headers.ticket_report_status_executor.address": "Address",
-  "excelExport.headers.ticket_report_status_executor.processing": "In progress",
-  "excelExport.headers.ticket_report_status_executor.completed": "Done",
-  "excelExport.headers.ticket_report_status_executor.canceled": "Canceled",
-  "excelExport.headers.ticket_report_status_executor.deferred": "Deferred",
-  "excelExport.headers.ticket_report_status_executor.closed": "Closed",
-  "excelExport.headers.ticket_report_status_executor.new_or_reopened": "Opened",
-  "excelExport.headers.ticket_report_status_property.address": "Address",
-  "excelExport.headers.ticket_report_status_property.processing": "In progress",
-  "excelExport.headers.ticket_report_status_property.completed": "Done",
-  "excelExport.headers.ticket_report_status_property.canceled": "Canceled",
-  "excelExport.headers.ticket_report_status_property.deferred": "Deferred",
-  "excelExport.headers.ticket_report_status_property.closed": "Closed",
-  "excelExport.headers.ticket_report_status_property.new_or_reopened": "New or reopened",
-  "excelExport.tickets.ticketCommentFilesCount": "files",
-  "excelExport.tickets.ticketFromResident": "resident",
-  "excelExport.tickets.ticketFromNonResident": "not resident",
-  "excelExport.sheetNames.buildings": "Properties",
-  "excelExport.sheetNames.tickets": "Tickets",
-  "billing.category.housing.name": "Rent",
-  "billing.category.repair.name": "Repairs",
-  "billing.category.electricity.name": "Electricity",
-  "billing.category.heating_hot_water.name": "Heating and hot water",
-  "billing.category.cold_water.name": "Cold water",
-  "billing.category.gas.name": "Gas",
-  "billing.category.trash.name": "Garbage collection",
-  "billing.category.parking.name": "Parking",
-  "billing.category.storage.name": "Storage space",
-  "billing.category.internet.name": "Internet",
-  "billing.category.radio.name": "Radio",
-  "billing.category.intercom.name": "Intercom",
-  "billing.category.telecom.name": "Telecom"
+    "Accept": "Accept",
```

Comment:
You need to change editor settings

Reply:
thanks 😅 fixed it

Reply:
you can also squash the commits so they don't contain large diffs. There will be only 1 commit with the necessary changes)

Reply:
nice, sounds useful, thanks!